File: 2.0/java/texts/findTextBroadcasts.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.campaigns.model.TextBroadcast;
import com.callfire.api.client.api.campaigns.model.request.FindTextBroadcastsRequest;
import com.callfire.api.client.api.common.model.Page;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        FindTextBroadcastsRequest request = FindTextBroadcastsRequest.create()
            .name("broadcast name")
            .label("my label")
            .running(true)
            .offset(0L)
            .limit(10L)
            .fields("items(id,name,status,message)")
            .build();
        Page<TextBroadcast> broadcasts = client.textBroadcastsApi().find(request);
    }
}

File: 2.0/csharp/texts/findTextBroadcasts.cs
using CallfireApiClient;
using CallfireApiClient.Api.Campaigns.Model;
using CallfireApiClient.Api.Campaigns.Model.Request;
using CallfireApiClient.Api.Common.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        var request = new FindBroadcastsRequest
        {
            Name = "broadcast name",
            Label = "my label",
            Running = true,
            Offset = 0,
            Limit = 10,
            Fields = "items(id,name,status,message)"
        };
        Page<TextBroadcast> broadcasts = client.TextBroadcastsApi.Find(request);
    }
}
